### ISSUE TITLE
JsonConvertor attribute prevents object population through GetEntriesByTypeAsync

### ISSUE THREAD

**Issue by deanebarker** [contentful/contentful.net#10]
I have a `Term` content type which has a reference to multiple `Chapter` content types through a `Chapters` field.  The POCO class for `Term` has this:

```
public List<Chapter> Chapters { get; set; }
```

Per the documentation, I have this on the `Chapter` class definition:

```
[JsonConverter(typeof(EntryFieldJsonConverter))]
public class Chapter
```

This works when getting `Term` entries -- I also get the referenced `Chapter` entries, populated correctly.

However, somewhere else in the code, I just get all the `Chapter` entries by themselves, like this:

```
var chapters = myContentfulClient.GetEntriesByTypeAsync<Chapter>("chapter").Result.ToList();
```

`chapters` ends up to be a list of 16 `null` values.  There _are_ 16 chapters, so it found them all, but it didn't populate the objects with any information.

On a hunch, I removed the `JsonConvertor` attribute from the `Chapter` class definition, and it worked again -- now I got all 16 `Chapter` entries, populated correctly.

But now my `Chapters` reference from `Term` doesn't work.

The existence of the `JsonConvertor` attribute makes one situation work, and the other not. What am I doing wrong?

**Comment by Roblinde** [comment]
Yeah, that's a nasty bug. I'll look into it asap.

**Comment by deanebarker** [comment]
I can work around it for now by just doing a search for the related chapter entries, but a fix would be nice.

**Comment by Roblinde** [comment]
A fix is published. A case I had overlooked completely, thanks for finding it. This should be resolved in 0.7.11-alpha that should be available through NuGet once it reindexes. Try it out and see if it sets you straight.

**Comment by deanebarker** [comment]
Yep, this worked. Thanks.
